lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -64,8 +70,7 @@ class Server {
     const requestUrl = parseURL(request.url);
     const filePath = requestUrl.pathname;
     const queryString = requestUrl.search && parseQueryString(requestUrl.search.slice(1));
-    let absoluteFilePath = path.join(__dirname, filePath);
-
+    let absoluteFilePath = path.join(this._baseDir, filePath);
```

Comment:
wanna take this opportunity to do this?
```suggestion
    let absoluteFilePath = `${this._baseDir}/${filePath}`;
```

Reply:
I don't 😆  haha 

While the normalization problems are quite rare, they do exist and are super annoying to uncover when they happen. Asking contributors to learn them, keep up with them, and risk being wrong one time when it matters seems worse IMO when `path.join` will definitely not lead you astray. I probably wouldn't ask people to replace `${}/${}` with `path.join` when I know it won't matter in a review, but I definitely wouldn't want to actively head in the opposite direction :)

Curious for your take though. If we do a poll I'll happily respect the team majority here 👍

Reply:
idk @brendankenny converted me to it but I did not absorb the dogma. :)

Reply:
survey done.

![image](https://user-images.githubusercontent.com/2301202/101087290-a3c66280-3577-11eb-8bbf-4e8f35213f2e.png)


 while preference for use of `path.join` was evenly split, the majority felt it shouldn't be converted in review one way or the other, so I'll leave these as-is :)